### ISSUE TITLE
PaginatorOptions Getters & Seters 

### ISSUE THREAD

**Issue by karamani** [NodaSoftLtd/TecdocSite#3]
По большому счёту, вкусовщина, но...

1) Геттеры и сеттеры рекомендуются ярыми адептами Php.

2) Код из цепочки сеттеров часто более нагляден, чем столбец с присвоениями.

**Comment by antflk** [comment]
Согласен. Но в данном случае пагинатор был скопирован из одной либы, и максимум на что я сподобился это дописать аннотации )
